tedana/io.py

```diff
@@ -259,8 +257,8 @@ def save_tsv(self, data, name):
         """
         if not isinstance(data, pd.DataFrame):
             raise TypeError(
-                "data must be a pandas dataframe, not type "
-                + str(type(data))
+                "data must be a pandas dataframe, not type " +
+                str(type(data))
```

Comment:
I don't think we need the `+`s in these.

Reply:
Would a format string be preferable?

Reply:
Haha, alright, I can fix that for function calls. Tests passed locally so I think we should be in the clear, if I can confirm we are here then I'll mark it ready for review.